src/main/scala/PcapExample.scala

```diff
@@ -69,14 +63,16 @@ object PcapExample {
         !(data + offsetBytes + n)
       }
       .foreach { v =>
-        native.stdio.printf(toCString("%02X"), v)
+        native.stdio.printf(c"%02X", v)
```

Comment:
Is this equivalent to toCString or is this what you use to represent a constant?

In which case line 79 also has a constant.